cli/command/checkpoint/cmd.go

```diff
@@ -16,7 +16,7 @@ func NewCheckpointCommand(dockerCli *command.DockerCli) *cobra.Command {
 			cmd.SetOutput(dockerCli.Err())
 			cmd.HelpFunc()(cmd, args)
 		},
-		Tags: map[string]string{"experimental": "", "version": "1.25"},
+		Tags: map[string]string{"experimental": "", "version": "1.26"},
```

Comment:
I think these determine the _minimum_ version that supports these commands, so should not be changed (same for the others below